setup.py

```diff
@@ -27,7 +27,7 @@
 if __name__ == '__main__':
     setup(
         name='parlai',
-        version=f'0.1.{DATE}{BUILD}',
+        version='0.1.{DATE}{BUILD}'.format(DATE=DATE, BUILD=BUILD),
```

Comment:
why no f string

Reply:
Because we've had two issues where people running python 3.5 tried to install and instead of getting the "sorry, this requires python >=3.6" message, they got a SyntaxError.